src/varsha/CharacterEx1.java

```diff
@@ -0,0 +1,22 @@
+package varsha;
+
+public class CharacterEx1 {
+
+	void printDigit(String input) {
+		int count = 0;
+		for (int index = 0; index < input.length(); index++) {
+			char ch = input.charAt(index);
+			boolean isflag = Character.isDigit(ch);
+			if (isflag == true) {
```

Comment:
You can write Character.isDigit(ch) direct here too

Reply:
Okay